packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -40,7 +40,7 @@ export interface IFabricTestLedgerV1ConstructorOptions {
  * Provides default options for Fabric container
  */
 const DEFAULT_OPTS = Object.freeze({
-  imageVersion: "2021-01-05-3400c06",
+  imageVersion: "latest",
```

Comment:
@jagpreetsinghsasan Please specify a tag that's designed to not change over time: This is to ensure that the CI/tests consistently succeed or fail while being executed with a given revision (git sha) of the code.